generators/client/templates/angular/src/main/webapp/app/admin/audits/audits.component.html.ejs

```diff
@@ -43,10 +43,10 @@
         <table class="table table-sm table-striped">
             <thead>
             <tr jhiSort [(predicate)]="predicate" [(ascending)]="reverse" [callback]="transition.bind(this)">
-                <th jhiSortBy="auditEventDate"><span jhiTranslate="audits.table.header.date">Date</span><fa-icon [icon]="'sort'"></fa-icon></th>
-                <th jhiSortBy="principal"><span jhiTranslate="audits.table.header.principal">User</span><fa-icon [icon]="'sort'"></fa-icon></th>
-                <th jhiSortBy="auditEventType"><span jhiTranslate="audits.table.header.status">State</span><fa-icon [icon]="'sort'"></fa-icon></th>
-                <th><span jhiTranslate="audits.table.header.data">Extra data</span></th>
+                <th id="auditEventDate" jhiSortBy="auditEventDate"><span jhiTranslate="audits.table.header.date">Date</span><fa-icon [icon]="'sort'"></fa-icon></th>
```

Comment:
I think it will not solve the issue. As per sonar docs:
> There are two ways of doing it:
>     Adding a scope attribute to <th> headers.
>     Adding an id attribute to <th> headers and a headers attribute to every <td> element.

I also prefer the first solution as that's recommended to be used.

Reply:
this would change from `<th id="auditEventDate"` to `<th scope="col"` right?

Reply:
yes, better to run analysis on local to confirm

Reply:
> this would change from `<th id="auditEventDate"` to `<th scope="col"` right?

@amatosg : Yes it will solve it. I've verified this on another project.